js/editor-ta.js

```diff
@@ -140,20 +157,13 @@ EditorTextArea.prototype.updateDimentions = function() {
   this.container_.style.height = (editorHeight) + 'px';
 }
 
-/**
- * Create an edit session for a new file. Each tab should have its own session.
- * @param {string} opt_content
- * @return {EditSession}
- */
-EditorTextArea.prototype.newSession = function(opt_content) {
-};
-
 /**
  * Change the current session, usually to switch to another tab.
- * @param {EditSession} session
+ * @param {SessionDescriptor} session
```

Comment:
Should the CodeMirror version of this method have its input type changed too?

Reply:
good pick up